eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FullText.php

```diff
@@ -231,6 +236,12 @@ public function handle(
         array $languageSettings
     ) {
         $subSelect = $query->subSelect();
+
+        $wordIdSubquery = $this->getWordIdSubquery($subSelect, $criterion->value);
+        if ($wordIdSubquery === null) {
+            return $query->expr->neq(1, 1);
```

Comment:
This is still not compatible with declared return type, and I think it's require few words of comment .